pkg/drivers/docker_driver.go

```diff
@@ -0,0 +1,371 @@
+// Copyright 2017 Google Inc. All rights reserved.
+
```

Comment:
Could you move these files using a "git mv" so history is preserved? It'll also make the review a lot smaller.

Reply:
If you look at each commit individually, the review will show just the differences between the two files like `git mv` would. For some reason viewing the whole PR review doesn't show this view, probably because Github is lagging since it's so big.